docs/Spark/Spark.md

```diff
@@ -1,10 +1,14 @@
 ---
-title: Spark
-id: Spark
+title: Copilot for Spark users
+id: copilot-for-spark-users
 description: Using Spark with Prophecy's Data Transformation Copilot
-tags: []
+tags: [spark, warehouse]
 ---
 
+Prophecy Data Transformation Copilot for Spark combines the best of both worlds; **high-quality code** based on software engineering best practices with a complete, **easy-to-use visual** environment.
+
+![CodeToVisual](./img/CodeToVisualSpark.png)
+
 ```mdx-code-block
 import DocCardList from '@theme/DocCardList';
 import {useCurrentSidebarCategory} from '@docusaurus/theme-common';
```

Comment:
When I click on the links in the "Standard Gems" list here, the page is not found: 
/Spark/gems/

Not sure if that's caused by this PR or a previous one.

Reply:
This might get fixed when we re-index the corpus after merging the PR.

Reply:
Okay, I'll check and re-index if needed.